injectors/linux.js

```diff
@@ -17,7 +20,27 @@ exports.getAppDir = async () => {
       '/opt/discord-canary',
       '/opt/DiscordCanary'
     ];
-    const discordPath = paths.find(path => existsSync(path));
+    let discordPath = paths.find(path => existsSync(path));
+
+    if (discordPath === undefined) {
+      const readlineInterface = readline.createInterface({
+        input: process.stdin,
+        output: process.stdout
+      });
+
+      console.log('A Discord Canary installation sdcould not be found at the usual paths.')
+       discordPath = await new Promise(resolve => readlineInterface.question('Provide your Discord Canary install location: ', customDiscordPath => {
+        if (existsSync(customDiscordPath)) {
+          readlineInterface.close();
+          resolve(customDiscordPath);
+        } else {
+          console.log('Path provided is invalid.');
```

Comment:
Should be more indicative that the installer is now going to close, instead of suddenly stopping